config/members-crossplane.yaml

```diff
@@ -790,3 +790,16 @@ spec:
     user: zzxwill
     organization: crossplane
     role: direct_member
+---
+apiVersion: organizations.github.crossplane.io/v1alpha1
+kind: Membership
+metadata:
+  name: crossplane-smileisak
+  labels:
+    org: crossplane
+spec:
+  forProvider:
+    inviteeId: 3457118
+    user: smileisak
```

Comment:
we've been keeping this list alphabetized by `user` field i think, could you insert this entry in that sorted order?

Reply:
@jbw976 fixed.